DOCS-7402 dedicated info (#201)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-aws.adoc

list the required and optional environment variables.

[NOTE]
====
Depending on your environment, you may need to modify environment variables specified within `fabric.tf` that are not 
listed in the tables below. This is common if you need to use existing AWS resources such as an existing VPC.

In Runtime Fabric, the inbound load balancer runs on each controller node in either shared or dedicated mode: 

> *Shared Mode* is the default setting and specifies the internal load balancer runs with the amount of CPU cores and memory you specify.

Reviewer: In the pages where we discuss installation steps, we should explain where the internal load balancer runs.

In shared mode, the internal load balancer is distributed across the controller nodes.

Author: Updated this - please see if I updated it as you intended - thank you!

> *Shared Mode* allows you to specify the amount of CPU cores and memory for the internal load balancer. In shared mode, the internal load balancer is distributed across the controller nodes. *Shared Mode* is the default setting. 

Reviewer 2: ditto GLOBAL formatting:
```suggestion
* *Shared Mode* enables you to specify the numbeer of CPU cores and amount of memory for the internal load balancer. 
In shared mode, the internal load balancer is distributed across the controller nodes. *Shared Mode* is the default setting. 
```

> *Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of CPU cores and memory.

Reviewer: In the pages where we discuss installation steps, we should explain where the internal load balancer runs.

In dedicated mode, the internal load balancer is deployed on the internal load balancer nodes.

Author: Updated this - please see if I updated it as you intended - thank you!

> *Dedicated Mode* specifies that all available resources are dedicated to the internal load balancer, so you cannot choose the amount of CPU cores and memory. In dedicated mode, the internal load balancer is deployed on dedicated internal load balancer nodes.

Reviewer 2: ```suggestion
* *Dedicated Mode* specifies that all available resources are dedicated to the internal load balancer, so you cannot choose the number of CPU cores and amount of memory. 
In dedicated mode, the internal load balancer is deployed on dedicated internal load balancer nodes.
```

If you experience an error related to AWS authorization, ensure you're using the same terminal window for 
verifying the variables and running the Terraform command.

. Run the script using Docker.
+
The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 
the Runtime Fabric update to Active status in Runtime Manager.
+
[NOTE]
This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.

> .. Navigate to the `../rtf-install-scripts/` directory. You must run terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, etc).

Reviewer 2: GLOBAL
Also, see if there is a way to promote this step and its peers, to avoid nesting.
```suggestion
.. Navigate to the `../rtf-install-scripts/` directory. 
You must run Terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, and so on).
```

> .. Initialize terraform (this only has to be performed once):

Reviewer 2: ditto promotion
```suggestion
.. Initialize Terraform (this only has to be performed once):
```

+
----
docker run -v $(pwd):/src -w /src/aws \
  -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN \
  hashicorp/terraform:0.12.6 init
----

> .. Copy the following to a text editor:

Reviewer 2: ditto

  -state=tf-data/rtf.tfstate
----

> .. Modify this using the data in the environment variables tables above.

Reviewer 2: "above" onscreen or in this doc? If the latter, use "shown earlier."

  -var mule_license='' \
  -state=tf-data/rtf.tfstate
----
+
The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 
the Runtime Fabric updates to Active status in Runtime Manager.
+
[NOTE]
====

> * This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.

Reviewer 2: If this is truly a list item
```suggestion
* This step installs Runtime Fabric across all servers to form a cluster. 
It might take 15 to 25 minutes or longer to finish.
```

> * The internal load balancer is distributed across the controller nodes for shared mode, and is deployed on the internal load balancer nodes for dedicated mode.

Reviewer 2: Why are these bullets?

> * The internal load balancer is distributed across the controller nodes for shared mode, and is deployed on the internal load balancer nodes for dedicated mode.

Reviewer 2: ditto
```suggestion
* The internal load balancer is distributed across the controller nodes for shared mode and is deployed on the internal load balancer nodes for dedicated mode.
```
